web/src/index.jsx

```diff
@@ -1,11 +1,12 @@
-import * as React from "react";
-import * as ReactDOM from "react-dom";
+import React from "react";
+import { createRoot } from "react-dom/client";
 import ReplicatedErrorBoundary from "./components/shared/ErrorBoundary";
 import Root from "./Root";
+const container = document.getElementById("app");
+const root = createRoot(container); // createRoot(container!) if you use TypeScript
 
-ReactDOM.render(
+root.render(
   <ReplicatedErrorBoundary>
     <Root />
-  </ReplicatedErrorBoundary>,
-  document.getElementById("app")
+  </ReplicatedErrorBoundary>
 );
```

Comment:
These are the main changes. I followed this guide.

Reply:
@rianfowler can you link the guide?

Reply:
forgot to do that sorry https://reactjs.org/blog/2022/03/08/react-18-upgrade-guide.html